nano/nano_node/entry.cpp

```diff
@@ -34,6 +34,8 @@ int main (int argc, char * const * argv)
 		("batch_size",boost::program_options::value<std::size_t> (), "Increase sideband batch size, default 512")
 		("debug_block_count", "Display the number of block")
 		("debug_bootstrap_generate", "Generate bootstrap sequence of blocks")
+		("debug_clear_online_weight", "clear online_weights table")
```

Comment:
We have cli --online_weight_clear in cli.cpp

Reply:
k will drop that we can use it instead, i must have missed it